src/enums/index.ts

```diff
@@ -13,6 +12,7 @@ export * from './filingNames'
 export * from './filingStatus'
 export * from './filingTypes'
 export * from './partyTypes'
+export * from './NameChangeOptions'
```

Comment:
I capitalized this to match the enum name itself. This is how we did it in our other UIs (mostly). We can update the rest of these later.